src/apps/base-images/jre/Dockerfile

```diff
@@ -4,3 +4,6 @@ LABEL maintainer="GeoServer PSC <geoserver-users@lists.sourceforge.net>"
 
 ENV JAVA_TOOL_OPTIONS=
 ENV JAVA_OPTS=
+
+# Install the system CA certificates for the JVM :wqnow that we're root
+RUN USE_SYSTEM_CA_CERTS=true /__cacert_entrypoint.sh
```

Comment:
where does this file come from? I can't find it

Reply:
```
docker run --rm eclipse-temurin:21-jre cat /__cacert_entrypoint.sh
```